hack/toolbox/server/Dockerfile.heavy

```diff
@@ -21,14 +21,14 @@ RUN apt-get install -y \
     net-tools \
     netcat \
     nmap \
-    python \
     python3 \
     ssh \
     sudo \
     tcpdump \
     traceroute \
+    unzip \
     vim \
-    wget
+    wget 
```

Comment:
nit: whitespace. You can change this to be automatically taken care of in your IDE